elm/Mpd.elm

```diff
@@ -134,11 +136,14 @@ wsMsgDecoder =
         , ( "database", Decode.succeed WSDatabase )
         ]
 
+
 float : Decode.Decoder Float
```

Comment:
oh since when do we have this? seems to be due to https://github.com/go-edn/edn/pull/18?

Reply:
Nice catch. It's the JSON->EDN commit: https://github.com/alicebob/siren/commit/caa285a701537a272e4ace27e6d0a763f008b3c6#diff-f3f2ba0c9505d433e7d7603a0c7e999eR137

Reply:
huh you'd think I'd remember putting in a work-around like that...